.github/workflows/check-codeowners.yaml

```diff
@@ -0,0 +1,17 @@
+name: Check CODEOWNERS
+
+on: pull_request
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: install yq
+        run: |
+          sudo snap install yq
```

Comment:
not sure how snap handles majo apllication updates inside same os version.
so i would suggest not to use the snap but to install yq via binary download, with a fixed version.

maybe we should also go for a fixed os version instead "ubuntu-latest".

Reply:
Unless we need pinned versions I'd say we keep it simple. 

- For GitHub actions it's safe to use `ubuntu-latest` unless a specific version for an application is required (ref: https://github.com/actions/virtual-environments)
- snap is the preferred method for installing yq on Ubuntu (https://github.com/mikefarah/yq#ubuntu-and-other-linux-distros-supporting-snap-packages). I notice it took about 20s, which isn't too bad for this use (we could wget/curl it, but then we'd incur the maintenance cost of keeping it up to date when the version doesn't matter for our use case). I haven't tested the official GitHub action, but this open issue and comment by the yq maintainer doesn't make me feel too motivated to recommend it yet: https://github.com/mikefarah/yq/issues/418

Reply:
It's now using the github action to download yq, which under the hood uses snap to install it ;-)

Reply:
Haha cool

Reply:
Ok, let's keep the snap.

I already had problems using Ubuntu latest as they just moved from Ubuntu 18.04 to 20.04. problems been Python related. So I guess this could be a problem again when the image switches to Ubuntu 22.04.